app/articles/server/methods/admin.js

```diff
@@ -0,0 +1,87 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { API } from '../utils/url';
+import { settings } from '../../../settings';
+
+const api = new API();
+
+function setupGhost(user, token, rcUrl) {
```

Comment:
Why do we need to pass rcUrl as a parameter here to this function, can't it be directly accessed here from some meteor method or settings?